### PR TITLE
[12.0] Fix non-English instances

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@CasVissers once this is merged, the issue you've encountered should be fixed
@pedrobaeza maybe we should add `text()=` as pre-merge check? or something like that?

**Comment by pedrobaeza** [comment]
Well, I'm not sure if there can be legit uses of `text()`.

**Comment by alexey-pelykh** [comment]
I guess no, in XML files.

**Comment by pedrobaeza** [comment]
Isn't there an easiest XPath expression? Look at Odoo source code. They add options without such difficulties...

**Comment by alexey-pelykh** [comment]
If they would add keys to groups. In next version it may be simplified, yet right now there are no way of locating needed group in shorter expression. Or I don't know one :)

File: hr_timesheet_role/report/report_timesheet_templates.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 <odoo>
     <!--
-        Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+        Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
         License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
     -->
 
     <template id="report_timesheet" inherit_id="hr_timesheet.report_timesheet">
-        <xpath expr="//th/span[text()='Responsible']/.." position="after">
+        <xpath expr="(//th)[count(//td/span[@t-field='l.user_id.partner_id.name']/../preceding-sibling::*)+1]" position="after">
```
Comment (alexey-pelykh) [inline hr_timesheet_role/report/report_timesheet_templates.xml:9]: @pedrobaeza "After column whose data is `partner_id.name`" - so no way to make this one simplier

**Review by pedrobaeza — APPROVED** [review]

File: hr_timesheet_task_required/views/res_config_settings.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="hr_timesheet.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@data-key='project']/h2[text()='Tasks Management']/following-sibling::div" position="inside">
+            <xpath expr="//div[@data-key='project']//field[@name='module_hr_timesheet']/../../.." position="inside">
```
Comment (alexey-pelykh) [inline hr_timesheet_task_required/views/res_config_settings.xml:13]: Well, I don't know how this may be simplified

Comment (alexey-pelykh) [reply hr_timesheet_task_required/views/res_config_settings.xml:13]: @pedrobaeza ^

**Comment by alexey-pelykh** [comment]
@CasVissers if possible, please check if this fixes issue for you and approve the PR if so

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM, thanks.

**Review by CasVissers-360ERP — APPROVED** [review]

**Pull request merged**